buch/teilnehmer.tex

```diff
@@ -6,13 +6,14 @@
 Julian Bärtschi,
 Jonas Gründler,
 Dominic Hüppi,
-Hansruedi Patzen%,
+Raphael Nestler%,
```

Comment:
Ich hab mich da mal eingetragen, auch wenn ich nur inoffizieller Teilnehmer war. Ist das in Ordnung so?